src/cheri/insns/amoswap_32bit_cap.adoc

```diff
@@ -15,8 +15,14 @@ Mnemonic::
 Encoding::
 include::wavedrom/amoswap_cap.adoc[]
 
-//{cheri_cap_mode_name} Description::
+Description::
 Atomic swap of capability type, authorized by the capability in `cs1`.
++
+The operation is equivalent to an atomically executed sequence of:
++
```

Comment:
This seems the wrong way around, we return the previous value, so LC first.

Reply:
yes good point!